bundles/org.openhab.binding.panamaxfurman/src/main/java/org/openhab/binding/panamaxfurman/internal/protocol/event/PanamaxFurmanConnectivityEvent.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.panamaxfurman.internal.protocol.event;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Marker interface for connectivity events
+ *
+ * @author Dave Badia - Initial contribution
+ */
+@NonNullByDefault
+public interface PanamaxFurmanConnectivityEvent {
```

Comment:
What is the use of this empty interface ?

Reply:
It's a marker interface which all connectivity event classes will implement.
Additional info about the marker interface pattern: https://www.baeldung.com/java-marker-interfaces